### DISCUSSION TITLE
Reddit

### DISCUSSION THREAD

**Discussion by ghost** [mienaiyami/yomikiru#23] [General]
The issue with my reddit account got resolved, I Immediately created a post recommending your application in [r/manhua ](https://www.reddit.com/r/Manhua/comments/yzayd8/offline_reader_for_manhwa_manhua/)and [r/manhwa](https://www.reddit.com/r/manhwa/comments/yzb5tb/offline_reader_for_manhwa_manhua/).

**Comment by ghost** [comment]
### In the future updates,

- Consider not adding anything that takes more ram and cpu usage like thumbnails, page pop-up animations, etc.
- These are all unnecessary clutter that increase the size of the application and take more resources of a pc, Imo. Which is unnecessary.
- The things that many users want are already present in your application.
- I mentioned almost every feature that many users wanted and you added them. There is nothing else after issue#18.
- If you find a way to decrease the cpu and ram usage further without giving up performance, do not forget to add it.

**Reply by mienaiyami** [reply]
>I also recommended your post in [r/manga](https://www.reddit.com/r/manga/comments/yzbzdp/offline_reader_for_manga_manhwa_manhua/?sort=new).

Thank you so much for being so supportive. Before you started using my app, I was mostly idle but wanted to code and improve my app, and since I thought I'm the only user I didn't have any motivation for it. I'm really grateful to you. I'm really out of ideas and if you want any app or web service, I will be happy to make it, free and open source (if its possible for me alone to make it) as it will help me improve as developer as well.

> Consider not adding anything that takes more ram and cpu usage like thumbnails, page pop-up animations, etc.

If this related to anything I added/have in my application. If so, please mention, I will improve on it soon.

>page pop-up animations

If you are referring to "Setting" popup, it only negligible amount of cpu and no ram. But you will notice a small peak in cpu usage for 2-5sec because opening settings loads features like theme and shortcut editor. 
___
Unrelated to all this, I recommend using ctrl+n for new window instead of launching app from executable.

**Reply by ghost** [reply]
> I also recommended your post in [r/manga](https://www.reddit.com/r/manga/comments/yzbzdp/offline_reader_for_manga_manhwa_manhua/?sort=new).

I do not know why, but it got too many downvotes, so I deleted the post in r/manga only. I am sorry.
The other posts are doing well with a 100% in r/manhwa and 80% in r/manhua with views totaling to 555 and 325 respectively and a total post shares of 6. I will not delete these two posts even if they do not perform good.

>I'm really grateful to you. 

Thank you very much.

>I'm really out of ideas and if you want any app or web service, I will be happy to make it, free and open source (if its possible for me alone to make it) as it will help me improve as developer as well.

Thank you very much. If I have any ideas I will suggest them to you.
> > Consider not adding anything that takes more ram and cpu usage like thumbnails, page pop-up animations, etc.
> 
> If this related to anything I added/have in my application. If so, please mention, I will improve on it soon.

This is not related to anything you have in your application.
I just wanted to convey that adding unnecessary things like thumbnails, page pop-up animations, etc will take more ram.

> > page pop-up animations
> 
> If you are referring to "Setting" popup, it only negligible amount of cpu and no ram. But you will notice a small peak in cpu usage for 2-5sec because opening settings loads features like theme and shortcut editor.

No. I do not mean that.
See below video to know what I meant by page preview pop-up animations.

https://user-images.githubusercontent.com/111482404/202859188-ad8e2b12-d240-4407-aa68-14d0194edee3.mp4

**Reply by mienaiyami** [reply]
>See below video to know what I meant by page preview pop-up animations.

 Oh okay, I intended to keep my app simple from start, so I wont add anything fancy which make it perform less.

**Comment by ghost** [comment]
The application in the above video is YACReader. One of Famous applications among manga community.

- In idle state 60 - 70 MB. When no manga is loaded. Only home screen.
- It takes 300-350 MB of ram in idle state, when a manga is opened.
- Once you start scrolling manga, the ram usage is between 500 - 600 MB.
- It is too much ram usage for 14MB rar size, Imo.

**Reply by mienaiyami** [reply]
Yep, it does is lot of ram for few mb images.

**Comment by ghost** [comment]
- ### So this has happened in r/manhwa.

![image](https://user-images.githubusercontent.com/111482404/203459534-cea61599-95b1-40ab-ada7-22e969dc82de.png)

- ### And The Reason they offered is,

![image](https://user-images.githubusercontent.com/111482404/203459433-cef9d3a0-e74a-4ec8-953e-b40d36c64586.png)

- They did not even offer a relevant reason. It has been 3 days since I reported the mods, but still no reply from them.
- I will create a new post again in r/manhwa after some time.
- By the way, the post in r/manhua is still live.

**Reply by mienaiyami** [reply]
>I will create a new post again in r/manhwa after some time.

You don't need to, otherwise it will get unnecessary hate.
But I really do appreciate that you are trying to advertise my app, thanks.
